cardinal/ecs/internal/testutil/testutil.go

```diff
@@ -36,7 +36,7 @@ func InitEngineWithRedis(t *testing.T, s *miniredis.Miniredis) *ecs.Engine {
 		Password: "", // no password set
 		DB:       0,  // use default DB
 	}, Namespace)
-	sm, err := ecb.NewManager(rs.Client)
+	sm, err := gamestate.NewEntityComponentBuffer(rs.Client)
```

Comment:
I've always understood this to be an "Entity Command Buffer". It buffers commands (i.e. state changes) related to entities.

Reply:
alright looks like I hallucinated that memory.